lib/qesapdeployment.pm

```diff
@@ -480,4 +485,50 @@ sub qesap_ansible_script_output {
     return $output;
 }
 
+=head3 qesap_create_aws_credentials
+
+    Creates a AWS credentials file as required by QE-SAP Terraform deployment code.
+=cut
+
+sub qesap_create_aws_credentials {
+    my ($key, $secret) = @_;
+    my %paths = qesap_get_file_paths();
+    my $credfile = script_output q|awk -F\" '/aws_credentials/ {print $2}' | . $paths{qesap_conf_trgt};
```

Comment:
is this to create a new path, starting from an existing one from the "paths catalogue" in **qesap_get_file_paths** ? If it is so, why not just to add the new path within **qesap_get_file_paths** in a more explicit way?

Reply:
> is this to create a new path, starting from an existing one from the "paths catalogue" in **qesap_get_file_paths** ? 

No. This is to create the credentials file in the path specified by the the YAML configuration file. 

> If it is so, why not just to add the new path within **qesap_get_file_paths** in a more explicit way?

Because this way ensures that if you run the test with a YAML configuration pointing to a location different than any default or anything specified on the settings, then the file will be created where both terraform and ansible are looking for it as the YAML configuration feeds the values for the ansible playbooks and the terraform configuration.

Reply:
Does it make sense to add a comment in the code like...?

```
# get the region from the conf.yaml or from a job setting if not present there
```

Is it the intent of this two lines?